spec/index.bs

```diff
@@ -3703,6 +3703,36 @@ enum GPUIndexFormat {
 };
 </script>
 
+The index format determines both the data type of index values in a buffer and, when used with
+strip primitive topologies ({{GPUPrimitiveTopology/"line-strip"}} or
+{{GPUPrimitiveTopology/"triangle-strip"}}) also specifies the primitive restart value. The
+<dfn dfn>primitive restart value</dfn> indicates which index value indicates that a new primitive
+should be started rather than continuing to construct the triangle strip with the prior two indexed
+vertices.
+
+{{GPURenderPipelineDescriptor}}s that specify a strip primitive topology must not leave the
```

Comment:
maybe "must not have" instead of "must not leave"?

Reply:
Fixed!